.travis.yml

```diff
@@ -23,21 +31,22 @@ addons:
     packages:
       - g++-4.9
 before_install:
-  - echo "$TRAVIS_NODE_VERSION"
-  - export "PATH=./node_modules/.bin:$PWD/node_modules/.bin:$HOME/.local/bin:$PATH"
   # coveralls
   - pip install --user cpp-coveralls
   # compilers
   - if [ "$CXX" = "g++" -a "$TRAVIS_OS_NAME" = "linux" ]; then export CXX="g++-4.9" CC="gcc-4.9" AR="gcc-ar-4.9" RANLIB="gcc-ranlib-4.9" NM="gcc-nm-4.9" ; fi
   - if [ "$CXX" = "clang++" ]; then export NPMOPT=--clang=1 ; fi
-  # node versions
-  - rm -rf ~/.nvm
-  - git clone --branch v0.33.2 https://github.com/creationix/nvm.git ~/.nvm
-  - source ~/.nvm/nvm.sh
-  - nvm install "$TRAVIS_NODE_VERSION"
-  - node --version
   - export CFLAGS="$CFLAGS -O3 --coverage" LDFLAGS="$LDFLAGS --coverage"
   - echo "CFLAGS=\"$CFLAGS\" LDFLAGS=\"$LDFLAGS\""
+  # nvs
+  - git clone --branch v$NVS_VERSION --depth 1 https://github.com/jasongin/nvs ~/.nvs
+  - . ~/.nvs/nvs.sh
+  - nvs --version
+  # node.js
+  - nvs add $NODEJS_VERSION
+  - nvs use $NODEJS_VERSION
+  - node --version
+  - npm --version
```

Comment:
Yes, just for reference when something fails. Most CI scripts I've seen do this.